rpc_state_reader/src/lib.rs

```diff
@@ -15,6 +16,14 @@ use starknet_in_rust::{
 use std::env;
 use thiserror::Error;
 
+#[cfg(test)]
+use cairo_vm::felt::felt_str;
+#[cfg(test)]
+use starknet_in_rust::{
+    definitions::constants::EXECUTE_ENTRY_POINT_SELECTOR,
+    transaction::{InvokeFunction, Transaction},
+};
```

Comment:
I don't know if I'm bullshitting here, but I think you can also do this this way:
```suggestion
#[cfg(test)]
use ::{
    cairo_vm::felt::felt_str,
    starknet_in_rust::{
        definitions::constants::EXECUTE_ENTRY_POINT_SELECTOR,
        transaction::{InvokeFunction, Transaction},
    },
};
```

Reply:
that would be nice, I'll check